src/transformState.js

```diff
@@ -4,8 +4,32 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
-function transformState(state, actions) {
-  // write code here
-}
+const transformState = (state, actions) => {
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear' :
+        for (const property in state) {
+          delete state[property];
+        }
+        break;
+
+      case 'removeProperties':
+        for (const property of action.keysToRemove) {
```

Comment:
[checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 3rd item
```suggestion
        for (const keyToRemove of action.keysToRemove) {
```